fe1-web/components/eventList/LaoProperties.tsx

```diff
@@ -10,21 +10,23 @@ import { useSelector } from 'react-redux';
 import ParagraphBlock from 'components/ParagraphBlock';
 import { Lao } from 'model/objects';
 import { makeCurrentLao } from 'store/reducers';
+import PropTypes from 'prop-types';
 import QRCode from '../QRCode';
 
-function renderProperties(lao: Lao) {
+function renderProperties(lao: Lao, url: string) {
   const creationDateString = lao.creation.timestampToString();
+  const value = lao.id.toString().concat(' ').concat(url);
```

Comment:
Reminder: there was a discussion this semester to put this in a JSON object and have it documented in JSON-Schema.
I don't remember where or when it happened, but I think the conclusion (robustness-wise) was a good one. 😉

Reply:
👌🏻I will change this.